app/src/ui/welcome/welcome.tsx

```diff
@@ -91,10 +91,10 @@ export class Welcome extends React.Component<IWelcomeProps, IWelcomeState> {
     }
 
     // Only advance when the state first changes...
-    if (this.props.signInState.kind !== nextProps.signInState.kind) {
+    if (this.props.signInState.kind === nextProps.signInState.kind) {
```

Comment:
Some day I'll learn how to computer

Reply:
![giphy-3](https://user-images.githubusercontent.com/13760/31897058-505b3e32-b7e3-11e7-801a-eee387b280e2.gif)

Reply:
![](https://user-images.githubusercontent.com/359239/31917391-ab3ca9aa-b89a-11e7-8171-9f8e29f96f15.gif)